src/main/target/NOX/target.h

```diff
@@ -44,9 +44,11 @@
 #define GYRO_1_CS_PIN           PB12
 #define GYRO_1_SPI_INSTANCE     SPI2
 
-#define USE_EXTI
-#define USE_GYRO_EXTI
-#define GYRO_1_EXTI_PIN         PA8
+// temporarily disable until bugfix is found
+//#define USE_EXTI
+//#define USE_GYRO_EXTI
+//#define GYRO_1_EXTI_PIN         PA8
+
```

Comment:
Legacy target man, you can do this on your own unified target config file. Just ``resource GYRO_CS 1 NONE``, you don't need to change the legacy target definition.

Reply:
Agreed

Reply:
See https://github.com/betaflight/unified-targets/pull/542

Reply:
Done.